app/code/Magento/GiftMessageGraphQl/Model/Resolver/Cart/Item/GiftMessage.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GiftMessageGraphQl\Model\Resolver\Cart\Item;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\GiftMessage\Api\ItemRepositoryInterface;
+
+/**
+ * Class provides ability to get GiftMessage for cart item
+ */
+class GiftMessage implements ResolverInterface
+{
+    /**
+     * @var ItemRepositoryInterface
+     */
+    private $itemRepository;
+
+    /**
+     * GiftMessageItem constructor.
+     *
+     * @param ItemRepositoryInterface $itemRepository
+     */
+    public function __construct(
+        ItemRepositoryInterface $itemRepository
+    ) {
+        $this->itemRepository = $itemRepository;
+    }
+
+    /**
+     * Return information about Gift message for item cart
+     *
+     * @param Field            $field
+     * @param ContextInterface $context
+     * @param ResolveInfo      $info
+     * @param array|null       $value
+     * @param array|null       $args
+     *
+     * @return array|Value|mixed
+     * @throws GraphQlInputException
+     * @throws NoSuchEntityException
```

Comment:
I may be wrong but it does not look like we have the case with `NoSuchEntityException` here. Could you remove this annotation, please?

Reply:
**Done**